.github/CONTRIBUTING.md

```diff
@@ -100,17 +100,25 @@ wilds-wdl-library/
 
 **The module folder must contain:**
 
-1. **`ww-toolname.wdl`** - Main WDL workflow file named for the tool it uses
-2. **`README.md`** - Comprehensive documentation
+1. **`ww-toolname.wdl`** - Main WDL file containing task definitions for the tool
+2. **`testrun.wdl`** - Test workflow demonstrating module functionality
```

Comment:
```suggestion
2. **`testrun.wdl`** - Test workflow demonstrating module functionality (must be named `testrun.wdl`)
```
Somehow make this explicitly clear, since the GHA python script looks for this file.